tasks/src/test/java/webdriver/google_cloud/page/GoogleCloudHomePage.java

```diff
@@ -11,6 +13,10 @@
 import org.openqa.selenium.support.ui.WebDriverWait;
 
 public class GoogleCloudHomePage extends GoogleCloudAbstractPage {
+  private final Logger logger = LogManager.getRootLogger();
+  private static final String PATTERN_FIND_TERM = "//b[contains(text(), '%s')]";
+  private static final int TIME_OUT_IN_SECONDS = 30;
```

Comment:
Better store all common timeouts inside a base page to be able to reuse them inside children